### ISSUE TITLE
Add example of an interactive 1D histogram

### ISSUE THREAD

**Issue by jbednar** [bokeh/datashader#225]
Datashader's aggregations are usually used as a generalization of a 2D histogram, but it should be straightforward to use the same code to get the data for a 1D histogram.  At the moment, doing so would need some minor tricks like adding a dummy constant column for the second dimension, but we could consider handling some of that behind the scenes.  

It would be particularly helpful to provide a full example of creating such a 1D histogram and then interactively visualizing it as a bar chart in Bokeh, so that a user could interactively zoom in to a region of a heavily biased histogram like:

  [1494889,      34,       1,       0,       1,       0,       0,     0,       0,       1]

to see more detail in the first bin.

**Comment by jbednar** [comment]
The biggest challenge will be that you can't just use InteractiveImage as usual with datashader, since you'll presumably want to see the histogram not as colors but as bar heights, so you'd need to write the equivalent of InteractiveImage but for a bar chart.  Such a class would need to request a new aggregation given the current x axis range, then copy the data from the aggregate array into a Bokeh ColumnDataSource and update the x axis values in it.  All fairly straighforward, but would take some work!

**Comment by poliu2s** [comment]
Any updates on this issue? This is a great feature that I hope to use.

**Comment by jbednar** [comment]
Since I opened this issue, we've added datashader support to [HoloViews](http://holoviews.org).  InteractiveImage is actually a tiny little bit of HoloViews that we pulled out long ago as an example of how to make dynamic datashader plots using Bokeh, and the full HoloViews package is much more flexible.  In HoloViews, it's fairly straightforward to make such a 1D dynamic histogram, and @philippjfr and I have now added an example of such a histogram to the last cell of the [NYC Taxi non-geographic notebook](https://anaconda.org/jbednar/nyc_taxi-nongeo).  This is definitely something that can be made simpler in future work, but it's already quite usable, so I'll close this issue.
